src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+                System.out.println("    Got it. I've added this task:\n" + "    " + whatToDo + "\n"
+                        + "    Now you have " + inputCount + " tasks in the list.");
+
+                input = in.nextLine();
+
+            } else if (input.contains("deadline")) {
+                Task whatDeadline = deadlineMethod(input);
+                System.out.println("    Got it. I've added this task:\n" + "    " + whatDeadline + "\n"
+                        + "    Now you have " + inputCount + " tasks in the list.");
+
+                input = in.nextLine();
+
+            } else if (input.contains("event")) {
+                Task whatEvent = eventMethod(input);
+                System.out.println("    Got it. I've added this task:\n" + "    " + whatEvent + "\n"
+                        + "    Now you have " + inputCount + " tasks in the list.");
+
+                input = in.nextLine();
+
+            } else { //adding in general orders
+                System.out.println("    added: " + input);
+                Task t = new Task(input);
+                tasks[inputCount] = t;
+                inputCount++;
+
+                input = in.nextLine();
+
+            }
+        }
+        System.out.println("    Bye! Hope to see you again soon!");
+
     }
+
+    private static String printList(Scanner in) {
+        String statusOfTask;
+        String input;
+        System.out.println("    Here are the tasks in your list:");
+
+        for (int outputCount = 0; outputCount < inputCount; outputCount++) {
+            statusOfTask = tasks[outputCount].getStatusIcon();
+            System.out.println("    " + (outputCount + 1) + "." + tasks[outputCount]);
+        }
+        input = in.nextLine();
+        return input;
+    }
+
+    private static String testTaskDone(String input, Scanner in) {
+        int taskNumber;
+        String[] inputSplitter;
+        inputSplitter = input.split(" ");
+        taskNumber = Integer.parseInt(inputSplitter[1]);
+        taskNumber = taskNumber - 1;
+
+        if (((taskNumber + 1) > 0) && ((taskNumber + 1) <= inputCount)) { //check for invalid inputs
+            input = taskDone(in, tasks[taskNumber]);
+
+        } else {
+            System.out.println("    Invalid input! Please check the list again!");
+            input = in.nextLine();
+        }
+        return input;
+    }
+
+    private static String taskDone(Scanner in, Task task) {
+        String statusOfTask;
+        String input;
+        System.out.println("    Nice! I've marked this task as done:" + System.lineSeparator());
+        System.out.println("    " + task + " has been updated to -->");
+
+        task.markAsDone(); //mark x in [ ]
+        statusOfTask = task.getStatusIcon();
+
+        System.out.println("    " + task);
+        input = in.nextLine();
+        return input;
+    }
+
+    public static Task toDoMethod(String input) {
+        String toDoDescription = input.substring(4).trim();
```

Comment:
We should avoid "magic numbers" like 4 here, you can define a CONST to replace 4